feat: add beagle flutter documentation (#717)
ZupIT/beagle-docs: content/en/resources/customization/beagle-for-flutter/services/http-client.md

---
title: HTTP Client
weight: 189
description: >-

>   You will find here information of how to configure and use the HTTP client in Beagle Flutter.

Reviewer: ```suggestion
In this section, you will information on how to configure and use the HTTP client in Beagle Flutter.
```
